stream-parent/stream-core/src/main/java/org/interledger/stream/frames/ConnectionAssetDetailsFrame.java

```diff
@@ -0,0 +1,71 @@
+package org.interledger.stream.frames;
+
+/*-
+ * ========================LICENSE_START=================================
+ * Interledger Core
+ * %%
+ * Copyright (C) 2017 - 2018 Hyperledger and its contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =========================LICENSE_END==================================
+ */
+
+import org.interledger.core.Immutable;
+
+import org.immutables.value.Value.Derived;
+
+/**
+ * A STREAM frame for propagating asset details for a Connection.
+ */
+public interface ConnectionAssetDetailsFrame extends StreamFrame {
+
+  /**
+   * Get the default builder.
+   *
+   * @return a {@link ConnectionAssetDetailsFrameBuilder} instance.
+   */
+  static ConnectionAssetDetailsFrameBuilder builder() {
+    return new ConnectionAssetDetailsFrameBuilder();
+  }
+
+  @Override
+  default StreamFrameType streamFrameType() {
```

Comment:
See #248 for unit test coverage tracking in this and all StreamFrame packets.

Reply:
Resolved in 2e252c2 (I thought I was working on #247 when I was doing this though)